.SRCINFO

```diff
@@ -1,15 +1,14 @@
 pkgbase = coursier
 	pkgdesc = Pure Scala Artifact Fetching
-	pkgver = 2.0.0_RC5_3
+	pkgver = 2.0.0_RC3_6
```

Comment:
This is probably a typo. It should be _RC5_6, right?

Reply:
Of course! I had to do some rebase and I might miss that :thinking:

Reply:
Updated!